### PR TITLE
Feature/#52 game init

### LINKED ISSUE #52: Logic/#52 - Game Init

### PR DESCRIPTION
**Pull request by vieraboschkova**
closes #52

### CODE REVIEW

**Review by paula0403 — DISMISSED** [review]
The code looks fine, only too much TO DO comments.

**Comment by vieraboschkova** [comment]
Part for preparing data for the herd to refactor in the next sprint.

File: src/app/logic/Game.ts [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+import { Bank } from './Bank';
+import { defaultGameConfiguration } from './defaultGameConfiguration';
+import { LivestockConfigInterface } from '../../Interfaces/LivestockConfigInterface';
+import { ProtectorsConfigInterface } from '../../Interfaces/ProtectorsConfigInterface';
+import { HerdOwners } from '../../Enums/HerdOwnerEnum';
+
+export class Game {
+  mode: GameModes;
+  roundTimeInSeconds: number;
+  // totalGameTimeInSeconds: number;
```
Comment (paula0403) [inline src/app/logic/Game.ts:20]: remove comment

File: src/app/logic/defaultGameConfiguration.ts [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+//     { name: AnimalNames.FOX, probability: 1 },
+//   ],
+//   [
+//     { name: AnimalNames.RABBIT, probability: 6 },
+//     { name: AnimalNames.SHEEP, probability: 3 },
+//     { name: AnimalNames.PIG, probability: 1 },
+//     { name: AnimalNames.COW, probability: 1 },
+//     { name: AnimalNames.FOX, probability: 1 },
+//   ],
+// ],
```
Comment (paula0403) [inline src/app/logic/defaultGameConfiguration.ts:136]: would You like to use this code from comments?

**Review by paula0403 — COMMENTED** [review]
Remove comments or use code from them.

File: src/app/logic/defaultGameConfiguration.ts (hunk 2 above)
Comment (vieraboschkova) [reply src/app/logic/defaultGameConfiguration.ts:136]: This code is supposed to be discussed when it comes to the way we create dice objects

File: src/app/logic/Game.ts (hunk 1 above)
Comment (vieraboschkova) [reply src/app/logic/Game.ts:20]: This a feature that might be introduced in the future

**Review by Enessetere — APPROVED** [review]
Looks good to me

**Review by paula0403 — APPROVED** [review]

**Pull request merged**